content/english/Kusto-KC7/activity-5.md

```diff
@@ -0,0 +1,14 @@
+---
+title: "Activity 4.5: KQL 101 Answers Explained"
```

Comment:
If we continue with the .5 naming scheme, I would suggest keeping the file names consistent.

activity-4-5.md, for example

Reply:
updated.